cmake/EthPolicy.cmake

```diff
@@ -20,4 +20,9 @@ macro (eth_policy)
 		# Allow selecting MSVC runtime library using CMAKE_MSVC_RUNTIME_LIBRARY.
 		cmake_policy(SET CMP0091 NEW)
 	endif()
+
+	# Avoid warnings in CMake 3.24.0:
```

Comment:
Well, that's obvious :) It would be more useful to say which warning this is preventing.

Reply:
Sorry, I've force-pushed this pull request, but the code is not changed, just add information about the warning to the comment.

Reply:
I'd say if we change this to `Prevent warning message related to changed default behavior of DOWNLOAD_EXTRACT_TIMESTAMP` it's good to merge